test/index.spec.js

```diff
@@ -78,11 +78,11 @@ describe('ReactCreditCards', () => {
       focused: 'number',
     });
 
-    expect(wrapper.find('.rccs__card').hasClass('rccs__card--amex')).toBe(true);
```

Comment:
The same thing here, we might need to revert these changes

Reply:
I'm not sure if I got your point here, but given the newly created `cardTypesMap`:

```js
export const cardTypesMap = {
  amex: ['amex', 'americanexpress', 'american-express'],
  dinersclub: ['diners', 'dinersclub', 'diners-club'],
  visaelectron: ['visaelectron', 'visa-electron'],
};
```

Should we use the object keys as our identifiers for the CSS class?